### DISCUSSION TITLE
Mod not installing

### DISCUSSION THREAD

**Discussion by EGImpleks** [TEHE-Studios/ExpandedHelicopterEvents#141] [Q&A]
After the last update, the modification stopped working on the server.
According to the instructions, I went and reinstalled the mod, now it just won't load. Help me please. Probably an off-topic question, but still.

Console log
WARN : Mod         , 1678875672733> 738,691> ZomboidFileSystem.loadModAndRequired> required mod "ExpandedHelicopterEvents" not found
![image](https://user-images.githubusercontent.com/96224735/225294178-81b8664b-fd76-44ae-aabc-d23fd4b79fff.png)

Mod added
![image](https://user-images.githubusercontent.com/96224735/225294382-745a0688-43a1-4077-8288-de1126f02478.png)
![image](https://user-images.githubusercontent.com/96224735/225294526-69bfbe7b-db80-420c-ae5d-793cd5ec7869.png)

But it doesn't want to upload to the folder
What can be done about it?
C:\servers\pzserver\steamapps\workshop\content\108600

**Comment by Chuckleberry-Finn** [comment]
The only mods that require EHE are the two sub-mods we've published, Super-Weird-Helis and Twitch-Integration for EHE. There was also another mod created by someone else adding Sheriff helicopters.

So my best guess would be that one of these mods is being loaded first.

**Comment by EGImpleks** [comment]
The mod works, everything is fine.
